src/rockstor/scripts/scheduled_tasks/reboot_shutdown.py

```diff
@@ -33,6 +38,45 @@ def validate_shutdown_meta(meta):
     return meta
 
 
+def all_devices_offline(addresses):
+    for address in addresses:
+        logger.debug('Pinging ' + address + '...')
```

Comment:
Simple style reformatting:
```suggestion
        logger.debug('Pinging {}...'.format(address))
```